### PR TITLE
Extract initialisation SQL into separate file

### PR DESCRIPTION
**Pull request by ryanslade**
This makes it easier to edit normally.

I think we may want to pick a SQL formatter for this which we run automatically too.

It looks like prettier already has a SQL formatter so maybe we can just use that since we already use it to format the json schema.

### CODE REVIEW

File: pkg/state/init.sql [hunk 1]
```diff
@@ -0,0 +1,340 @@
+CREATE SCHEMA IF NOT EXISTS placeholder;
+
+CREATE TABLE IF NOT EXISTS placeholder.migrations
```
Comment (ryanslade) [inline pkg/state/init.sql:3]: `placeholder` will be replaced in code.

File: pkg/state/state.go [hunk 2]
```diff
@@ -411,7 +71,8 @@ func (s *State) Init(ctx context.Context) error {
 	}
 
 	// Perform pgroll state initialization
-	_, err = tx.ExecContext(ctx, fmt.Sprintf(sqlInit, pq.QuoteIdentifier(s.schema), pq.QuoteLiteral(s.schema)))
```
Comment (ryanslade) [inline pkg/state/state.go:414]: We weren't using the output of `pq.QuoteLiteral`

**Review by andrew-farries — APPROVED** [review]
This looks good 👍 

I think we should also find a solution for SQL formatting, either as part of this PR or a follow up.

**Comment by ryanslade** [comment]
> I think we should also find a solution for SQL formatting, either as part of this PR or a follow up.

I'll see if I can get it to use Prettier

**Comment by ryanslade** [comment]
@andrew-farries PTAL

**Review by andrew-farries — APPROVED** [review]

**Review by kvch — APPROVED** [review]

**Pull request merged**